pkg/controller/cassandradatacenter/statefulset.go

```diff
@@ -165,7 +223,7 @@ func newSysctlLimitsContainer(cdc *cassandraoperatorv1alpha1.CassandraDataCenter
 		Image:           cdc.Spec.CassandraImage,
 		ImagePullPolicy: cdc.Spec.ImagePullPolicy,
 		SecurityContext: &corev1.SecurityContext{
-			Privileged: func() *bool { b := true; return &b }(),
+			Privileged: boolRef(cdc.Spec.PrivilegedSupported),
```

Comment:
Can't this just be `&cdc.Spec.PrivilegedSupported`? The old lambda was to work around that you can'd to `&true`.

Reply:
:man_facepalming: